bitbots_blackboard/src/bitbots_blackboard/capsules/pathfinding_capsule.py

```diff
@@ -17,11 +17,18 @@ def __init__(self):
         self.orientation_threshold = rospy.get_param('behavior/body/pathfinding_orientation_threshold')
         self.pathfinding_pub = None  # type: rospy.Publisher
         self.pathfinding_cancel_pub = None  # type: rospy.Publisher
+        self.ball_obstacle_active_pub = None
         self.goal = None  # type: PoseStamped
         self.current_pose = None # type: PoseStamped
+        self.status = -1 # Current status of movebase
+        self.ACTIVE = 1
+        self.CANCELED = 2
+        self.SUCCEEDED = 3
```

Comment:
Please use an enum or a dict or something

Reply:
Especially the part below, where you compare the message's status to these three values shows that you should just import `actionlib_msgs.msg.GoalStatus` and use `GoalStatus.ACTIVE`, `GoalStatus.PREEMPTED` (i.e. the goal was aborted because a new goal was sent, this is what you call canceled), and `GoalStatus.SUCCEEDED`. Maybe there should also a behavior for `GoalStatus.ABORTED` which happens when move base itself aborts the action for another reason than a newer goal.